src/components/atoms/UiButton/UiButton.vue

```diff
@@ -4,20 +4,29 @@
     v-keyboard-focus
     v-bind="routeAttrs"
     class="ui-button"
+    :tabindex="isLoading ? -1 : null"
```

Comment:
`isLoading` (and also maybe `--is-disabled` should become `isDisabled`) should make button `aria-disabled`.